jest.config.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
         '<rootDir>/.vscode/',
         '<rootDir>/public/',
     ],
-    testEnvironment: 'jsdom',
+    testEnvironment: './fixJSDOMEnvironment.ts',
```

Comment:
This was required because `jsdom` was failing with errors for missing `structuredClone` function (introduced in node v17). The error caused all unit tests to fail.